src/cpp/session/SessionModuleContext.cpp

```diff
@@ -348,6 +348,17 @@ SEXP rs_rstudioEdition()
 
 // get version
 SEXP rs_rstudioVersion()
+{
+   std::string processedVersion = RSTUDIO_VERSION;
```

Comment:
Would it be easier and/or less fragile to compose this from the component variables (RSTUDIO_VERSION_MAJOR, etc.)?

Reply:
We can do that - we'll still have to apply regex on the `RSTUDIO_VERSION_SUFFIX` portion to strip the release channel and/or `pro` from the string